dashboard/test/ui/features/star_labs/ailab.feature

```diff
@@ -0,0 +1,67 @@
+Feature: AI Lab
+
+  @as_student
+  Scenario: Training a model with an in-house dataset and importing it into App Lab
+    Given I am on "http://studio.code.org/s/allthethings/lessons/43/levels/1"
+    And I wait to see ".editor-column"
+    And I see the dynamic instructions are showing "uitest-dynamic-instruction-selectDataset" key
```

Comment:
You can move the common `uitest-dynamic-instruction-` part of the identifier to the function implementation, so that only the `selectDataset` part of the key needs to be passed in.

Reply:
Much cleaner, thanks!